manifests/postgresql_settings.pp

```diff
@@ -16,13 +16,13 @@
   Optional[Float[0,1]] $reports_autovacuum_vacuum_scale_factor  = 0.01,
   Boolean    $manage_reports_autovacuum_cost_delay     = true,
   String     $maintenance_work_mem                     = $all_in_one_pe_install ? {
-                                                           false => "${::memory['system']['total_bytes'] / 1024 / 1024 / 3}MB",
-                                                           true  => "${::memory['system']['total_bytes'] / 1024 / 1024 / 8}MB",
-                                                         },
+    false => "${::memory['system']['total_bytes'] / 1024 / 1024 / 3}MB",
+    true  => "${::memory['system']['total_bytes'] / 1024 / 1024 / 8}MB",
+  },
```

Comment:
Is this really a readability win?  I don't think I agree with puppet-lint on this warning